.rubocop.yml

```diff
@@ -731,9 +731,12 @@ Layout/IndentationStyle:
 Layout/IndentationWidth:
   Width: 1
 
+Metrics/MethodLength:
+  Max: 25
+
```

Comment:
The default is 10 but I think 25 is a more realistic length as it fits on the regular terminal size and we wouldn't have to ignore it.